q2_types/feature_data/_format.py

```diff
@@ -404,7 +404,9 @@ def validate(self, *args):
 class ProteinFASTAFormat(FASTAFormat):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.alphabet = "ABCDEFGHIKLMNPQRSTVWXYZ*"
+        upper_case = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+        lower_case = "abcdefghijklmnopqrstuvwxy"
```

Comment:
It looks like lowercase `z` was left out here - based on your summary I think you probably intended to include that.

Reply:
I did. Thanks for catching that.